src/zenodo_client/api.py

```diff
@@ -65,6 +74,15 @@ def download_zenodo_latest(deposition_id: str, path: str, force: bool = False, *
     return Zenodo(**kwargs).download_latest(deposition_id, path=path, force=force)
 
 
+def drop_none(d):
+    """Drop (nested) entries with None value."""
+    if isinstance(d, list):
```

Comment:
I'm not super happy with this implementation. Maybe `exclude_none` from https://pydantic-docs.helpmanual.io/usage/exporting_models/ would do the trick and we don't have to implement anything

Reply:
plus ORCID is required so that should also be fixed

Reply:
> plus ORCID is required so that should also be fixed

it is not required from the Zenodo side, cf. https://developers.zenodo.org/#representation; the `Creator` dataclass also permits leaving it at its default, `None`.

Reply:
wait i am dumb, this doesn't use pydantic haha